packages/server/src/service/org/passport.ts

```diff
@@ -237,5 +237,16 @@ export const registerPassportEndpoints = (app: Express.Application) => {
     }
   )
 
+  app.get('/embed', validSession, async (req, res) => {
```

Comment:
Embed should be tied to a landing page and that should be in the URL.  Let's just put this on the details page (e.g. `dashboard/foobar`).